module.txt

```diff
@@ -1,9 +1,9 @@
 {
     "id" : "SnakeTournament",
     "version" : "0.2.0",
-    "author" : "",
+    "author" : "Josharias, rzats, TheJYKoder",
```

Comment:
Neither you nor me should be here. README/metadata updates don't _really_ count as authorship.